### PR TITLE
Fix create bug when publish failed

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: src/Angor/Client/Pages/Create.razor [hunk 1]
```diff
@@ -427,6 +425,8 @@
             return;
         }
 
+        project.ProjectInfo.CreationTransactionId = signedTransaction.GetHash().ToString();
```
Comment (DavidGershony) [inline src/Angor/Client/Pages/Create.razor:428]: You also need to handle the new state in line 266 when checking on loading the page

Comment (dangershony) [reply src/Angor/Client/Pages/Create.razor:428]: This is the result from the relay, what state do we need to handle here?

Comment (DavidGershony) [reply src/Angor/Client/Pages/Create.razor:428]: We should only navigate to the view page if creation transaction id is populated

Comment (dangershony) [reply src/Angor/Client/Pages/Create.razor:428]: Gotcha ill fix it

**Review by DavidGershony — APPROVED** [review]

File: src/Angor/Client/Pages/Create.razor [hunk 2]
```diff
@@ -252,18 +253,71 @@
...
+
+                        project.ProjectInfo = nostrProject;
+                    }
+
+                    if (applicationDataOnNostr && nostrProfileCreated)
+                    {
+                        var findProject = storage.GetFounderProjects().FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == project.ProjectInfo.ProjectIdentifier);
+                        if (findProject != null)
+                        {
+                            if (string.IsNullOrEmpty(findProject.CreationTransactionId))
```
[the remark this answers is not in the document]
Comment (dangershony) [reply src/Angor/Client/Pages/Create.razor:309]: As agreed lets do it in 3 stages, this will come in a following PR

**Pull request merged**